internal/controller/kyma/controller.go

```diff
@@ -354,9 +360,9 @@ func (r *Reconciler) handleProcessingState(ctx context.Context, kyma *v1beta2.Ky
 
 	if r.WatcherEnabled() {
 		errGroup.Go(func() error {
-			if err := r.SKRWebhookManager.Install(ctx, kyma); err != nil {
+			if err := r.SKRWebhookManager.Reconcile(ctx, kyma); err != nil {
 				r.Metrics.RecordRequeueReason(metrics.SkrWebhookResourcesInstallation, queue.UnexpectedRequeue)
-				if errors.Is(err, &watcher.CertificateNotReadyError{}) {
+				if errors.Is(err, certificate.ErrSkrCertificateNotReady) {
```

Comment:
so this error need to be designed within `watcher` package, but not in `certificate`.

Reply:
Follow-up: https://github.com/kyma-project/lifecycle-manager/issues/2396